CHANGELOG.asciidoc

```diff
@@ -3,6 +3,152 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.17.0]]
+=== Beats version 7.17.0
+https://github.com/elastic/beats/compare/v7.16.3...v7.17.0[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Change Docker base image from CentOS 7 to Ubuntu 20.04 {pull}29681[29681]
+- Add job.name in pods controlled by Jobs {pull}28954[28954]
+- Enrich kubernetes metadata with node annotations. {pull}29605[29605]
+
+*Auditbeat*
+
+*Filebeat*
+
+*Heartbeat*
+
+- Fix broken macOS ICMP python e2e test. {pull}29900[29900]
+- Only add monitor.status to browser events when summary. {pull}29460[29460]
+- Also add summary to journeys for which the synthetics runner crashes. {pull}29606[29606]
+- Update size of ICMP packets to adhere to standard min size. {pull}29948[29948]
```

Comment:
Hey @ChrsMark, it seems like none of these are actually breaking changes. We might have made the mistake of appending to the wrong section throughout this last push for 7.17.

Just to be sure it would be good if either @andrewvc or @vigneshshanmugam could just confirm.